Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -125,7 +125,8 @@
   id: UplinkEmag
   category: Utility
   itemId: Emag
-  description: This sequencer, or "emag", can be used to open or hack a huge variety of items around the station.
+  listingName: Eletromagnetic Card
+  description: The business card of the syndicate, this sequencer is able to break open airlocks and tamper with a variety of station devices. Moderate cooldown between usages.
```

Comment:
i dont think this is even true, it just has a recharge so if you use them all up  you have to wait a bit